cdap-common/src/main/java/io/cdap/cdap/common/conf/Constants.java

```diff
@@ -2242,6 +2242,9 @@ public enum Direction {
       OUTGOING,
       BOTH
     }
+
+    public static final String FIELD_LINEAGE_EMISSION_ENABLED =
+        "system.cdap.field.lineage.emission.enabled";
```

Comment:
I don't think you need `system.cdap` prefixes. Also we have most of the configuration in `metadata.messaging.`, so it makes sense to name it .e.g "metadata.messaging.lineage.field.emission.enabled". Also please put default "true" into cdap-defaults.xml

Reply:
Ok if we are putting default in cdap-defaults, then we can use cConf.getBoolean()